app/helpers/metrics_helper.rb

```diff
@@ -0,0 +1,101 @@
+module MetricsHelper
+  def parse_agent_data(agents)
+    agents.map do |a|
+      {
+        x: a.label,
+        y: a.count
+      }
+    end
+  end
+
+  def sum_rollups(rollups)
+    rollups.map { |r| r[:count] }.reduce(:+)
+  end
+
+  def interval_options
+    Rollups::HourlyDownload::INTERVALS.map { |i| [I18n.t(".helpers.label.metrics.interval.#{i.downcase}"), i] }
```

Comment:
some constants were added into `Rollups::HourlyDownload` and `Rollups::DailyUnique` to represent options. these are also now translated in `en.yml`